database/src/main/scala/cromwell/database/migration/metadata/table/SymbolTableMigration.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.database.migration.metadata.table
+
+import java.io.{IOException, ByteArrayInputStream}
+import java.sql.{PreparedStatement, ResultSet}
+import java.util.zip.GZIPInputStream
+
+import cromwell.database.migration.metadata.{MetadataMigration, MetadataStatement}
+import liquibase.database.jvm.JdbcConnection
+import liquibase.exception.CustomChangeException
+import org.apache.commons.codec.binary.Base64
+import org.apache.commons.io.IOUtils
+import wdl4s.types.{WdlPrimitiveType, WdlType}
+import wdl4s.values._
+
+import scala.util.{Failure, Success, Try}
+
+abstract class SymbolTableMigration extends MetadataMigration {
+  override protected def migrateRow(connection: JdbcConnection, statement: PreparedStatement, row: ResultSet, idx: Int): Unit = {
+    // Try to coerce the value to a WdlValue
+    val value = for {
+      wdlTypeValue <- Try(row.getString("WDL_TYPE"))
+      wdlType <- Try(WdlType.fromWdlString(wdlTypeValue))
+      wdlValueValue <- Try(row.getString("WDL_VALUE"))
+      inflated <- inflate(wdlValueValue)
+
+      wdlValue <- wdlType match {
+        case p: WdlPrimitiveType => p.coerceRawValue(inflated)
```

Comment:
Unfortunately that this distinction in behavior needs to be in the caller; it would be nice if `WdlPrimitive`s overrode `fromWdlString` to do `coerceRawValue`.